FutureNorthants/northants-website
# Fix for table not displaying slices

Use replace to renderParagraph if it is a `drupal-paragraph` for tables. Add test with embedded paragraphs in a table. 

## To Test
- Edit a page and add a table in the body
- Add a slice within a table cell
- Save and publish the page
- View the page in the frontend and now the slice should appear within the table
- Run component tests with `npx cypress run-ct`
